cpp/assetmanager.cpp

```diff
@@ -52,9 +42,9 @@ Texture *AssetManager::load_texture(const std::string &name) {
 		// to display the tex as soon at it exists.
 
 		// return the big X texture instead
-		ret = this->missing_tex;
+		ret = get_missing_tex();
```

Comment:
Changed for the sake of consistancy with the old code inside WITH_INOTIFY. Pointless imho.